### PR TITLE
[RFC] Are we aiming to be able to write plot_montage like this?

### PR DESCRIPTION
**Pull request by massich**
This is just me thinking outloud.

While working in #6639 I find myself touching things that make 
montages look more similar. And I think we should be able to get
to something similar to what this PR shows.

This requires:
- unifyied attribute names
- no name dups in montagew

cross-ref: #6461

cc: @agramfort

### CODE REVIEW

File: mne/viz/montage.py [hunk 1]
```diff
@@ -27,47 +25,25 @@ def plot_montage(montage, scale_factor=20, show_names=True, kind='topomap',
...
-            n_dupes = dupes.shape[0]
-            idx = np.setdiff1d(montage.selection, dupes[:, 1]).tolist()
-            logger.info("{} duplicate electrode labels found:".format(n_dupes))
-            logger.info(", ".join([ch_names[d[0]] + "/" + ch_names[d[1]]
-                                   for d in dupes]))
-            logger.info("Plotting {} unique labels.".format(n_chans - n_dupes))
-            montage.ch_names = [montage.ch_names[i] for i in idx]
-            ch_names = montage.ch_names
-            montage.pos = montage.pos[idx, :]
-            montage.selection = np.arange(n_chans - n_dupes)
```
Comment (agramfort) [inline mne/viz/montage.py]: this was here for a reason. cc @larsoner 

+1 for unify attributes of montage and digmontage. point_names is not great. I also think that only channels have names.

Comment (larsoner) [reply mne/viz/montage.py]: Indeed, `git blame` points to #4592 and #4403

**Comment by agramfort** [comment]
this code will die when we remove Montage class after the next release.

**Comment by massich** [comment]
sure, but I would not have close it. Since it was showing how the plot (for both montage, and Digmontage) was aimed.

None the less I think that just cross-referencing all these would be enough

cross-referencing: #6412, #6744

**Pull request closed without merging**